api/foreign.go

```diff
@@ -1021,6 +1278,13 @@ func (entity *ForeignBusinessVentures) Valid() (bool, error) {
 	return entity.List.Valid()
 }
 
+// ClearNos clears the "no" answers on application rejection
+func (entity *ForeignBusinessVentures) ClearNos() error {
+	entity.HasForeignVentures.ClearNo()
```

Comment:
probably out of the scope of this PR since we want it in today, but an idea I had for abstracting this is:
- if we could have a generic property name that references the same value as `HasForeignVentures` for `ForeignBusinessVentures` or `HasForeignVoting` for `ForeignBusinessVoting`, then these methods could potentially all be abstracted to a single source.

Reply:
You can see how I did something like that for the TestClearBasicSectionNos test. I could see us trying something similar to implement all the basics. I think that if we did that we'd probably end up recording them all in the reject.go file instead of split out. As is I kind of like that we succinctly describe what needs to be cleared right next to the actual struct.